### DISCUSSION TITLE
电子书翻译中英文分离

### DISCUSSION THREAD

**Discussion by bradwang888** [immersive-translate/immersive-translate#1738] [Ideas]
### 新功能 | New Feature

翻译电子书导出选择双语导出时为了提高阅读效率高，能否做到“所翻译出来的中文全部在上面、英文原文全部在下面”，而不是现在这样“英文一段、中文一段”？

### 为什么您想要添加这些功能 | Why you want to add these features

提高阅读效率

**Comment by theowenyoung** [comment]
暂时无法做到。
